test/integration/ComputeExamples.test.ts

```diff
@@ -0,0 +1,703 @@
+/// - **Git**. Instructions for installing Git can be found here: https://git-scm.com/book/en/v2/Getting-Started-Installing-Git
+/// - **Node.js** can be downloaded from here: https://nodejs.org/en/download/
+/// - **Docker** can be installed from here: https://docs.docker.com/get-docker/. Please note that Docker must run as a non-root user, you can set this up by following these instructions: https://docs.docker.com/engine/install/linux-postinstall/
+
+/// ## 1. Initialize services
+
+/// Ocean.js uses off-chain services for metadata (Aquarius) and consuming datasets (Provider).
+
+/// We start by initializing the services. To do this, we clone the Barge repository and run it. This will run the current default versions of [Aquarius](https://github.com/oceanprotocol/aquarius), [Provider](https://github.com/oceanprotocol/provider), and [Ganache](https://github.com/trufflesuite/ganache) with [our contracts](https://github.com/oceanprotocol/contracts) deployed to it.
+
+/// ```bash
+/// git clone https://github.com/oceanprotocol/barge.git
+/// cd barge/
+/// ./start_ocean.sh --with-provider2 --no-dashboard --with-c2d
+/// ```
+
+/// ## 2. Create a new Node.js project with TypeScript
+
+/// Start by creating a new Node.js project. Open a new terminal and enter the following commands:
+
+/// ```bash
+/// mkdir compute-quickstart
+/// cd compute-quickstart
+/// npm init
+/// # Answer the questions in the command line prompt
+/// touch compute.ts
+/// # On linux press CTRL + D to save
+/// ```
+
+/// Next, we need to setup our TypeScript compiler options. Create a new file called `tsconfig.json` in the root of the `compute-quickstart` directory.
+
+/// ```bash
+/// touch tsconfig.json
+/// # Copy the following json content into the file, On linux press CTRL + D to save
+/// ```
+
+/// ```json
+/// {
+///   "compilerOptions": {
+///     "lib": ["es6", "es7"],
+///     "module": "CommonJS",
+///     "target": "ES5",
+///     "esModuleInterop": true,
+///     "allowSyntheticDefaultImports": true,
+///     "outDir": "./dist/",
+///     "declaration": true,
+///     "declarationDir": "./dist/"
+///   },
+///   "include": [
+///     "compute.ts"
+///   ],
+///   "exclude": [ "node_modules", "dist" ]
+/// }
+/// ```
+
+/// Now you can compile your TypeScript project.
+/// If you have TypeScript installed use the following command:
+
+/// ```bash
+/// tsc
+/// ```
+
+/// If you don't have TypeScript installed you can install it using the command below and then compile using the above command:
+
+/// ```bash
+/// npm install -g typescript
+/// ```
+
+/// Or if you don't want to install TypeScript use the following command to compile your file:
+/// ```bash
+/// npx tsc compute.ts
+/// ```
+
+/// To run your script as we go along, compile the script then you can use the following command:
+
+/// ```bash
+/// node dist/compute.js
```

Comment:
Am I right in thinking we need to export the contract addresses before we run the file?

```
export ADDRESS_FILE="${HOME}/.ocean/ocean-contracts/artifacts/address.json"
```